src/dotnet/CarbonAware.WebApi.Tests/unitTests/CarbonAwareControllerTests.cs

```diff
@@ -0,0 +1,79 @@
+namespace CarbonAware.WepApi.UnitTests;
+
+using System.Collections.Generic;
+using CarbonAware.Model;
+using CarbonAware.WebApi.Controllers;
+using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+
+/// <summary>
+/// Testing Web API Controller
+/// </summary>
+[TestFixture]
+public class CarbonAwareControllerTests : TestsBase
+{   
+    /// <summary>
+    /// Tests that successfull call to plugin results in action with 200 status
+    /// </summary>
+    [Test]
+    public async Task SuccessfulCallReturnsOk()
```

Comment:
Will this test always work because it's using hte static JSON controller? Is there some way to indicate that here?